src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/registration/display_vetting_types.html.twig

```diff
@@ -66,9 +66,9 @@
                 <h3>{{ 'ss.registration.vetting_type.title.ra_vetting'|trans }}</h3>
                 <hr>
                 <p>{{ 'ss.registration.vetting_type.description.ra_vetting'|trans }}</p>
-                <form action="{{ url('ss_registration_send_registration_email', {'secondFactorId': secondFactorId}) }}" method="get">
+                <a href="{{ url('ss_registration_send_registration_email', {'secondFactorId': secondFactorId}) }}">
                     <button type="submit" class="btn btn-primary">{{ 'ss.registration.vetting_type.button.ra_vetting'|trans }}</button>
```

Comment:
This construction is not 100% yet from a html structure point of view.

I'd say: remove the `<button>` element but keep the button text as the anchor text for the `<a>` element. And style the `<a>` to make it look like a button. I think we already have utility (css) classes ready for that.

Would you be so kind to look into that?

Reply:
Sorry, I spent the entire afternoon but I can't get it to look visually the same without the button-tags. Leaving it 'as-is', but feel free to try it yourself